src/api/attachments.js

```diff
@@ -28,11 +28,29 @@ Object.assign(AttachmentsApi.prototype, {
     create: smoochMethod({
         params: ['source', 'access'],
```

Comment:
should `access` be renamed here? How does this work with the object syntax?

i.e.

```javascript
smooch.attachments.create({
   appId: '',
   access: {
      access: 'public'
      for: 'message',
      appUserId: 'xyz'
   }
})
```

???

Reply:
Ahhh that's a good point. I'll test it and add a unit test.